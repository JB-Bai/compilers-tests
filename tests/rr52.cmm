struct JxvKD F05DqGEj(int er7CC_GP[794], int y08uYL8[287]){
	9986;
}
struct DQDjK3Jb SwmFErjrV;
float L5674ZXxN, K26C14HfCP[073077][0xEFf6][0xD2D][0xB2], ADC_G0;

