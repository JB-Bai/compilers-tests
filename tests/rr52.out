Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        StructSpecifier (1)
          STRUCT
          Tag (1)
            ID: JxvKD
      FunDec (1)
        ID: F05DqGEj
        LP
        VarList (1)
          ParamDec (1)
            Specifier (1)
              TYPE: int
            VarDec (1)
              VarDec (1)
                ID: er7CC_GP
              LB
              INT: 794
              RB
          COMMA
          VarList (1)
            ParamDec (1)
              Specifier (1)
                TYPE: int
              VarDec (1)
                VarDec (1)
                  ID: y08uYL8
                LB
                INT: 287
                RB
        RP
      CompSt (1)
        LC
        StmtList (2)
          Stmt (2)
            Exp (2)
              INT: 9986
            SEMI
        RC
    ExtDefList (4)
      ExtDef (4)
        Specifier (4)
          StructSpecifier (4)
            STRUCT
            Tag (4)
              ID: DQDjK3Jb
        ExtDecList (4)
          VarDec (4)
            ID: SwmFErjrV
        SEMI
      ExtDefList (5)
        ExtDef (5)
          Specifier (5)
            TYPE: float
          ExtDecList (5)
            VarDec (5)
              ID: L5674ZXxN
            COMMA
            ExtDecList (5)
              VarDec (5)
                VarDec (5)
                  VarDec (5)
                    VarDec (5)
                      VarDec (5)
                        ID: K26C14HfCP
                      LB
                      INT: 30271
                      RB
                    LB
                    INT: 61430
                    RB
                  LB
                  INT: 3373
                  RB
                LB
                INT: 178
                RB
              COMMA
              ExtDecList (5)
                VarDec (5)
                  ID: ADC_G0
          SEMI
